master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -1064,10 +1064,14 @@ private[celeborn] class Master(
   private[master] def handleCheckQuota(
       userIdentifier: UserIdentifier,
       context: RpcCallContext): Unit = {
-    val userResourceConsumption = handleResourceConsumption(userIdentifier)
-    val (isAvailable, reason) =
-      quotaManager.checkQuotaSpaceAvailable(userIdentifier, userResourceConsumption)
-    context.reply(CheckQuotaResponse(isAvailable, reason))
+    if (conf.quotaEnabled) {
```

Comment:
IMO quotaEnabled should only control quotaManager check quota or not，Resource-related metrics should be independent.

Reply:
Done